ui_src/src/components/createStationForm/index.js

```diff
@@ -166,9 +166,6 @@ const CreateStationForm = ({ createStationFormRef, getStartedStateRef, finishUpd
                 case 3:
                     replicas = ['No HA (1)', 'HA (3)'];
                     break;
-                case 5:
```

Comment:
It should be dynamic. The broker needs to set the available_replicas to three, which will be handled in the UI.
You don't need to delete it.